.github/workflows/main.yml

```diff
@@ -126,7 +126,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-        version: ['v2.2.6', 'v2.3.3', 'v2.4.5']
+        version: ['v2.2.6', 'v2.3.3', 'v2.4.5', 'v2.4.6']
```

Comment:
Do we still want to test on 2.4.5? We've only been testing on the latest point release.

Reply:
Just for a short while, to keep testing enterprise release compatibility.